### PR TITLE
Add numpy-groupies recipe

### PR DESCRIPTION
**Pull request by andersy005**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by andersy005** [comment]
@conda-forge/staged-recipes, this is ready for review

File: recipes/numpy-groupies/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    - numpy
+
+test:
+  imports:
+    - numpy_groupies
+
+about:
+  home: https://github.com/ml31415/numpy-groupies
+  license: BSD
+  license_family: BSD
```
Comment (chrisburr) [inline recipes/numpy-groupies/meta.yaml:36]:
```suggestion
  license: BSD-2-Clause
```

**Comment by ocefpaf** [comment]
We already had it: https://github.com/conda-forge/numpy_groupies-feedstock

It is a `-` vs `_` case :-(

**Comment by andersy005** [comment]
> It is a - vs _ case :-(

Hahaha... Good catch, @ocefpaf! 

The `anaconda.org/search` api misled me :) I wish the search api wasn't too strict when it comes to `_` and `-`

**Comment by isuruf** [comment]
Thanks @chrisburr for removing the recipe.

**Comment by ocefpaf** [comment]
@andersy005 it happens to all of us. We need to do multiple outputs like: https://github.com/conda-forge/pre-commit-feedstock/blob/master/recipe/meta.yaml to fix this.

**Comment by chrisburr** [comment]
Urg, sorry for missing that and thanks for noticing @ocefpaf. I think I managed to delete the recipe and cancel the CI before it did anything that matters. 🤞 

@andersy005 FYI: There is a quite nice search at https://conda-forge.org/feedstocks/. It only searched feedstock names so it isn't perfect but I often find it useful.

**Comment by isuruf** [comment]
I wonder why the linter didn't warn

**Comment by jakirkham** [comment]
Is the linter looking for variants on the name? Thought it was using exact match (at least that's how it worked in the past). Not to say it shouldn't be change, just reflecting on why it didn't warn.

**Comment by chrisburr** [comment]
The package name was an exact match wasn't it? The folder had a dash but the meta.yaml has an underscore.

**Comment by andersy005** [comment]
> @andersy005 FYI: There is a quite nice search at conda-forge.org/feedstocks. It only searches feedstock names so it isn't perfect but I often find it useful.

👍  That's awesome.. I wasn't aware of this...

**Comment by beckermr** [comment]
Yes the linter should have caught this. :/

**Comment by synapticarbors** [comment]
I thought https://github.com/conda-forge/conda-smithy/blob/master/conda_smithy/lint_recipe.py#L692-L696 would have caught it.

**Comment by jakirkham** [comment]
> The package name was an exact match wasn't it? The folder had a dash but the meta.yaml has an underscore.

Sorry maybe I misread. Though based on Joshua's point, it sounds like it shouldn't have mattered if this was off.

Maybe the next step is try adding a test based on this to the linter and see if it fails. If it does, we can debug and fix the problem. If it doesn't fail, something else is going on...

**Pull request merged**